From 6e14220b5aaac953fae2757939e33cb7f0dc3421 Mon Sep 17 00:00:00 2001
From: Mario Behn <mario.behn@gmail.com>
Date: Fri, 23 Oct 2026 15:02:37 +0200
Subject: [PATCH 03/14] Pluto and Pluto+: Linux 6.12.114 from a patched ADI
 kernel branch

Kernel now comes from mbehn1976/linux branch tezuka/6.12.114-xlnx-2026r1: ADI xlnx/release/v6.12.y-2026r1 (b486fe8) with the stable releases 6.12.78 to 6.12.114 merged, the six Tezuka patches applied in the tree, and one fix for the AXI DAC data select (cf_axi_dds: attach the buffer setup ops again), which stops the TX tone that kept playing after the last writer closed. BR2_LINUX_KERNEL_PATCH is dropped for these two boards. Other boards keep the ADI pin and the patch directory.

Signed-off-by: Mario Behn <mario.behn@gmail.com>
---
 configs/pluto_maiasdr_defconfig     | 3 +--
 configs/plutoplus_maiasdr_defconfig | 3 +--
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/configs/pluto_maiasdr_defconfig b/configs/pluto_maiasdr_defconfig
index 64a2be2..4af68f1 100644
--- a/configs/pluto_maiasdr_defconfig
+++ b/configs/pluto_maiasdr_defconfig
@@ -15,7 +15,6 @@ BR2_RELRO_FULL=y
 BR2_FORTIFY_SOURCE_2=y
 BR2_TIME_BITS_64=y
 BR2_TARGET_OPTIMIZATION="-mfpu=neon -ftree-vectorize -funsafe-math-optimizations"
-BR2_LINUX_KERNEL_PATCH="${BR2_EXTERNAL_PLUTOSDR_PATH}/board/tezuka/common/patches/linux/adi-6.12.y"
 BR2_GLOBAL_PATCH_DIR="${BR2_EXTERNAL_PLUTOSDR_PATH}/board/tezuka/common/patches"
 BR2_TARGET_GENERIC_HOSTNAME="pluto"
 BR2_TARGET_GENERIC_ISSUE="Welcome to Tezuka"
@@ -24,7 +23,7 @@ BR2_TARGET_GENERIC_ROOT_PASSWD="analog"
 BR2_TARGET_GENERIC_GETTY_PORT="ttyPS0"
 BR2_LINUX_KERNEL=y
 BR2_LINUX_KERNEL_CUSTOM_TARBALL=y
-BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="https://github.com/analogdevicesinc/linux/archive/b486fe824008ce25f56036b03d891063e6f233a8.tar.gz"
+BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="https://github.com/mbehn1976/linux/archive/678c4de70fdf40a490a574403537ced2e82cdba6.tar.gz"
 BR2_LINUX_KERNEL_USE_CUSTOM_CONFIG=y
 
 BR2_LINUX_KERNEL_DTS_SUPPORT=y
diff --git a/configs/plutoplus_maiasdr_defconfig b/configs/plutoplus_maiasdr_defconfig
index 928ea84..02ec178 100644
--- a/configs/plutoplus_maiasdr_defconfig
+++ b/configs/plutoplus_maiasdr_defconfig
@@ -15,7 +15,6 @@ BR2_RELRO_FULL=y
 BR2_FORTIFY_SOURCE_2=y
 BR2_TIME_BITS_64=y
 BR2_TARGET_OPTIMIZATION="-mfpu=neon -ftree-vectorize -funsafe-math-optimizations"
-BR2_LINUX_KERNEL_PATCH="${BR2_EXTERNAL_PLUTOSDR_PATH}/board/tezuka/common/patches/linux/adi-6.12.y"
 BR2_GLOBAL_PATCH_DIR="${BR2_EXTERNAL_PLUTOSDR_PATH}/board/tezuka/common/patches"
 BR2_TARGET_GENERIC_HOSTNAME="plutoplus"
 BR2_TARGET_GENERIC_ISSUE="Welcome to Tezuka"
@@ -24,7 +23,7 @@ BR2_TARGET_GENERIC_ROOT_PASSWD="analog"
 BR2_TARGET_GENERIC_GETTY_PORT="ttyPS0"
 BR2_LINUX_KERNEL=y
 BR2_LINUX_KERNEL_CUSTOM_TARBALL=y
-BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="https://github.com/analogdevicesinc/linux/archive/b486fe824008ce25f56036b03d891063e6f233a8.tar.gz"
+BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="https://github.com/mbehn1976/linux/archive/678c4de70fdf40a490a574403537ced2e82cdba6.tar.gz"
 BR2_LINUX_KERNEL_USE_CUSTOM_CONFIG=y
 
 BR2_LINUX_KERNEL_DTS_SUPPORT=y
-- 
2.53.0

